script.js

```diff
@@ -30,3 +29,108 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
   }
 });
+
+// projects-modal-popup
+const projects = [
+  {
+    id: 0,
+    backImg: 'images/pj-1.svg',
+    projectName: 'Professional Art Printing Data',
+    projectNote:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    projectStackUsed: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    id: 1,
+    backImg: 'images/project2.svg',
+    projectName: 'Data Dashboard Healthcare',
+    projectNote:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    projectStackUsed: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    id: 2,
+    backImg: 'images/pj3.png',
+    projectName: 'Website Portfolio',
+    projectNote:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    projectStackUsed: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    id: 3,
+    backImg: 'images/pj-1.svg',
+    projectName: 'Professional Art Printing Data',
+    projectNote:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    projectStackUsed: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    id: 4,
+    backImg: 'images/project2.svg',
+    projectName: 'Data Dashboard',
+    projectNote:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    projectStackUsed: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    id: 5,
+    backImg: 'images/pj3.png',
+    projectName: 'Website Portfolio ',
+    projectNote:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    projectStackUsed: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+];
+
+const projectCards = document.querySelector('.project-cards');
+
+projects.forEach((project, i) => {
+  const projectElement = document.createElement('div'); // div for each project = projectElement
+  projectElement.classList.add('card', `project${i + 1}`);
+
+  // Create HTML structure for each project
+  projectElement.innerHTML = `
+    <div class="project-name">
+      <p class="name">${project.projectName} </p>
+      
+    </div>
+    <p class="project-note">${project.projectNote}</p>
+    <div class = "project-stack">
+      <ul class="project-stack-used">
+        ${project.projectStackUsed
+    .map((tech) => `<li class="tech-used">${tech}</li>`)
+    .join('')}          
```

Comment:
- [ ] I will implore you to make the buttons witdth and height for the mobile version to be as the Figma.

### Your project
![image](https://github.com/H-Ramalan/portfolio-desktop/assets/86200356/99d3feca-5e3e-4d19-919b-e5df7a19e60b)

### Figma
![image](https://github.com/H-Ramalan/portfolio-desktop/assets/86200356/53e95cd0-3c3c-4abc-ac01-4029bec612bf)